src/Umbraco.Examine/ContentValueSetBuilder.cs

```diff
@@ -58,8 +88,8 @@ public override IEnumerable<ValueSet> GetValueSets(params IContent[] content)
                     {"urlName", urlValue?.Yield() ?? Enumerable.Empty<string>()},                  //Always add invariant urlName
                     {"path", c.Path?.Yield() ?? Enumerable.Empty<string>()},
                     {"nodeType", c.ContentType.Id.ToString().Yield() ?? Enumerable.Empty<string>()},
-                    {"creatorName", (c.GetCreatorProfile(_userService)?.Name ?? "??").Yield() },
-                    {"writerName",(c.GetWriterProfile(_userService)?.Name ?? "??").Yield() },
+                    {"creatorName", (creatorIds.TryGetValue(c.CreatorId, out var creatorProfile) ? creatorProfile.Name : "??").Yield() },
+                    {"writerName", (writerIds.TryGetValue(c.CreatorId, out var writerProfile) ? writerProfile.Name : "??").Yield() },
```

Comment:
@Shazwazza shouldn't it be `(writerIds.TryGetValue(c.WriterId, out var writerProfile)` here instead of `c.CreatorId`?

Reply:
I think so @bjarnef